app/models/miq_server/worker_management/monitor.rb

```diff
@@ -40,21 +40,10 @@ def worker_not_responding(w)
   end
 
   def sync_workers
-    result = {}
-    MiqWorkerType.worker_class_names.each do |class_name|
-      begin
-        c = class_name.constantize
-        raise NameError, "Constant problem: expected: #{class_name}, constantized: #{c.name}" unless c.name == class_name
-
-        result[c.name] = c.sync_workers
-        result[c.name][:adds].each { |pid| worker_add(pid) unless pid.nil? }
-      rescue => error
```

Comment:
Actually got a little too eager deleting code, this isn't just rescuing the NameError but also anything from `c.sync_workers` let me put some of this back